.github/workflows/build_conda_macos.yml

```diff
@@ -160,15 +160,18 @@ jobs:
             echo "${{ inputs.repository }}/${SMOKE_TEST_SCRIPT} not found"
           else
             echo "${{ inputs.repository }}/${SMOKE_TEST_SCRIPT} found"
-
-            ${CONDA_RUN} conda install -y pytorch -c pytorch cpuonly
-
             CONDA_BINARY="$(ls ${{ inputs.repository }}/dist/${ARCH_NAME}/*.tar.bz2)"
 
             ${CONDA_RUN} conda install -v -y -c pytorch-nightly pytorch
             ${CONDA_RUN} conda install --offline "$CONDA_BINARY"
 
-            ${CONDA_RUN} conda install -y 'ffmpeg<5'
+            if [[ "${{ inputs.package-name }}" = "torchaudio" ]]; then
+              ${CONDA_RUN} conda install -y 'ffmpeg<5'
```

Comment:
Shouldn't this be a part of the repository specific verification scripts?

Reply:
Repo-specific validation scripts are currently just python scripts. I'm going to add shell scripts that install the necessary run-time deps for running the smoke tests soon, this is just a stop-gap.